src/internal/kopia/merge_details.go

```diff
@@ -4,13 +4,18 @@ import (
 	"github.com/alcionai/clues"
 
 	"github.com/alcionai/corso/src/internal/common/prefixmatcher"
+	"github.com/alcionai/corso/src/internal/connector/graph/metadata"
 	"github.com/alcionai/corso/src/pkg/backup/details"
 	"github.com/alcionai/corso/src/pkg/path"
 )
 
 type DetailsMergeInfoer interface {
 	// ItemsToMerge returns the number of items that need to be merged.
 	ItemsToMerge() int
+	// TODO(keepers): remove once sharepoint fully supports metadata files.
+	// ItemsToMerge returns the number of items that need to be merged,
+	// excluding any item that ends in .meta or .dirmeta
+	ItemsToMergeSansMeta() int
```

Comment:
can you expand on the need for this? We shouldn't be adding metadata files to backup details at all by this point and we shouldn't be pulling them forward from base snapshots

Reply:
SP should also already support metadata files since those are now where file names are stored. It just doesn't support permissions right now